Revit_Core_Engine/Convert/Structure/FromRevit/MaterialFragment.cs

```diff
@@ -37,6 +39,24 @@ public static partial class Convert
         /****               Public Methods              ****/
         /***************************************************/
 
+        [Description("Converts a Revit Material to an Structure.MaterialFragments.IMaterialFragment.")]
+        [Input("material", "Revit Material to be converted.")]
+        [Input("settings", "Revit adapter settings to be used while performing the convert.")]
+        [Input("refObjects", "Optional, a collection of objects already processed in the current adapter action, stored to avoid processing the same object more than once.")]
+        [Output("material", "Structure.MaterialFragments.IMaterialFragment resulting from converting the input Revit Material.")]
+        public static IMaterialFragment MaterialFragmentFromRevit(this Material material, RevitSettings settings = null, Dictionary<string, List<IBHoMObject>> refObjects = null)
+        {
+            return material.MaterialFragmentFromRevit(null, settings, refObjects);
```

Comment:
Yeah agreed this looks a bit artificial. The reason why I added the new overloads without `grade` parameter is because the `ConvertMethod` query searches only through methods that have the following:
- return type of `IBHoMObject` or a collection of `IBHoMObjects`
- 1st input parameter of type `Element`
- 2nd input parameter of type `RevitSettings`
- 3rd input parameter of type `Dictionary<string, List<IBHoMObject>>`

This is needed to make sure that on invocation of the method the number of passed parameters is equal to the number of inputs. I will draw up a flowchart very soon to visualise the whole process.